third_party/move/extensions/async/move-async-vm/tests/sources/AccountStateMachine.v2_exp

```diff
@@ -0,0 +1,55 @@
+publishing Actor
```

Comment:
Same here. Once we switch to v2 compiler as the default, those things can be updated. This does not give us relevant new test coverage, most test cases here and in the EVM code are not specific to Move features, but to the tool.